.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
Please don't commit files from the `.idea` folder.

Reply:
Hello, sorry about that, I have removed the .idea folder